products/migrations/0002_auto_20210917_1028.py

```diff
@@ -1,54 +0,0 @@
-# Generated by Django 3.2.7 on 2021-09-17 10:28
```

Comment:
Same is the case here, we don't delete migrations that are shipped, there is a concept regarding squashing migrations but that would be something we do later on. for now let's not delete migrations that were not added through this PR and which are already part of main.

Reply:
I understand now. But now the files have been deleted and restoring them creates more errors among dependencies. I will keep it mind in future